fornax.py

```diff
@@ -65,46 +100,71 @@ def download(self):
         info = self.process_data_info()
         return download_file(info['access_url'])
 
+    
+class AWSDataHandlerException(Exception):
+    pass
 
 
 class AWSDataHandler(DataHandler):
     """Class for managaing access to data in AWS"""
 
-    def __init__(self, product, profile=None, **kwargs):
+    def __init__(self, product, access_url_column='access_url', profile=None, requester_pays=False):
```

Comment:
We should use keyword only for all optional parameters, it will make life down the line easier if/when we need to extend the API, etc.

```suggestion
    def __init__(self, product, *, access_url_column='access_url', profile=None, requester_pays=False):
```